pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -545,6 +545,58 @@
     patches = [ ./patches/coq_nvim/emulate-venv.patch ];
   };
 
+  cord-nvim =
+    let
+      version = "2024-07-19";
+      src = fetchFromGitHub {
+        owner = "vyfor";
+        repo = "cord.nvim";
+        rev = "cd97c25320fb0a672b11bcd95d8332bb3088ecce";
+        hash = "sha256-66NtKteM1mvHP5wAU4e9JbsF+bq91lmCDcTh/6RPhoo=";
+      };
+      rustPackage = rustPlatform.buildRustPackage {
+        pname = "cord.nvim-rust";
+        inherit version src;
+
+        cargoSha256 = "sha256-6FYf4pHEPxvhKHHPmkjQ40zPxaiypnpDxF8kNH+h+tg=";
+
+        buildPhase = ''
+          cargo build --release --lib
+        '';
```

Comment:
You have manually overriden the `buildPhase` to add `--lib` ?
If so, you might be able to simply pass it using `cargoBuildFlags` ?

Reply:
Done, I had no idea about cargoBuildFlags.